_pages/how-we-work/tools/anyconnect.md

```diff
@@ -2,46 +2,10 @@
 title: AnyConnect
 ---
 
-AnyConnect is a VPN client that we use to connect to GSA's intranet.
-
-## Setup
-
-### VPN Client
-Your machine may already have AnyConnect installed. If it is, it will be under Cisco>AnyConnect in your Applications.
-
-If it is not installed, here is how you can get access:
-
-1. Request the AnyConnect VPN Client via [ServiceNow](https://gsa.service-now.com/sp?id=sc_cat_item&sys_id=1bfdfdca78d3a400ce3ddff91a64940b&sysparm_category=b628125b7cec0100a6e757fe35f45fb8). Someone from GSA IT will reach out to you within the day.
-2. Follow their directions to install AnyConnect.
-
-Your installation screen should look like this:
-
-![AnyConnect Installation Screen]({{site.baseurl}}/images/anyconnect/1.png)
-
-## Logging into Cisco AnyConnect
-
-  1. Click on Launchpad (Rocket ship icon in the Dock at the bottom of your screen) 
-  2. Click on the Cisco AnyConnect Mobility Client icon to launch
-  3. Select your server in the pop-up window and click connect
-  4. Enter ENT login and password
-  5. Go to [otp.gsa.gov](http://otp.gsa.gov) and enter your ENT info to get the Tokencode
-     - You can also obtain a Tokencode by installing [SecureAuth OTP](https://preview-insite.gsa.gov/employee-resources/information-technology/do-it-yourself-self-help/telework-technology/secureauth/install-secureauth-otp-on-ios-apple)) on your phone, as described in the [Get a one-time password?]({{site.baseurl}}/distributed/#get-a-one-time-password) handbook section
-  6. Enter the Tokencode back into thee AnyConnect pop-up screen and submit
-
-## Usage
-
-Refer to [Insite](https://preview-insite.gsa.gov/employee-resources/information-technology/do-it-yourself-self-help/telework-technology/virtual-private-network-vpn/vpn-to-mac-os-quick-setup#connect) for the latest instructions on how to use the AnyConnect VPN Client. 
```

Comment:
Aidan, is there any way we can be sure this older link either 404s or redirects to the newer PIV way of doing things?

Reply:
Yep, all Mac documentation in InSite is [being phased out](https://docs.google.com/a/gsa.gov/document/d/1Dz9otyQRi-PFCJVjq5p5A6pbqTKOO-_VIW0TD-Mc5j4/edit?disco=AAAADcLt9Pg).